public/js/event-views.js

```diff
@@ -327,6 +331,103 @@ views.DialogView = Backbone.Marionette.Layout.extend({
                 break;
         }
     },
+
+    submitContactInfo: function(event) {
+        event.preventDefault(); 
+        
+        var preferredContact = {}; //Making an object for preferred contact 
+
+        var noShareVal = false;
+        var scope = $("#my-contact-info-modal");
+        var emailInfo = $("[name=email_info]", scope).val();
+        var twitterHandle = $("[name=twitter_handle]", scope).val();
+        var linkedinURL = $("[name=linkedin_url]", scope).val();
+        var noShareChkBox = $("[name=no-share]", scope).is(':checked');
+
+        //Hiding the error classes
+        $(".empty-contact-info-error", scope).hide();
+
+        $(".email-validate-error").hide();
+        $(".twitter-validate-error").hide();
+        $(".linkedin-validate-error").hide();
```

Comment:
Suggestion: use a common error class for all error messages (e.g. ``.error-msg``), and simplify this call to ``$(".error-msg", scope).hide();``.

Otherwise: add scope to the email/twitter/linkedin selectors too.